Lecture 5 corrections (and possible correction to Lecture 3)

EvZissel opened the pull request:
In Bellman-Ford algorithm - policy update step

EvZissel committed:
Merge pull request #1 from avivt/master

pull sync

EvZissel committed:
Lecture 3 - possible correction

line 156 - I think it should be d[u] without tilde

EvZissel committed:
Lecture 5 - corrections

1. line 31 - grammar 
2. lines 125-126 - Fixed policy value iteration for discounted return using vector notation: missing 𝛾 - the discount factor 
3. line 209 - whitespace
4. line 327 - punctuation

avivt commented on Lecture3.tex:
```diff
@@ -153,7 +153,7 @@ \subsection{The Bellman-Ford Algorithm}
       \tab{if  $d[v] < \infty $}
 
-      \tab{\tab{set $\pi [v] \in \arg {\min _u}\{ w(v,u) + \tilde d[u]:(v,u) \in E\} $}}
+      \tab{\tab{set $\pi [v] \in \arg {\min _u}\{ w(v,u) + d[u]:(v,u) \in E\} $}}
```
This is actually OK as it is

EvZissel replied:
@avivt I agree - I restored the notation

avivt commented on Lecture5.tex:
```diff
@@ -122,8 +122,8 @@ \section{The Fixed-Policy Value Function}\label{s:FP_VF}
-          \[{V_n}{\kern 1pt}  = {r^\pi } + {P^\pi }{r^\pi } +  \ldots  + {({P^\pi })^{n - 1}}{r^\pi } + {({P^\pi })^n}{V_0} = \sum\limits_{t = 0}^{n - 1} {{{({P^\pi })}^t}{r^\pi }}  + {({P^\pi })^n}{V_0}.\]
-Similarly,    ${V^\pi } = \sum\limits_{t = 0}^\infty  {{{({P^\pi })}^t}{r^\pi }}$.
+          \[{V_n}{\kern 1pt}  = {r^\pi } + {\gamma}{P^\pi }{r^\pi } +  \ldots  + {({\gamma}{P^\pi })^{n - 1}}{r^\pi } + {({\gamma}{P^\pi })^n}{V_0} = \sum\limits_{t = 0}^{n - 1} {{{({\gamma}{P^\pi })}^t}{r^\pi }}  + {({\gamma}{P^\pi })^n}{V_0}.\]
+Similarly,    ${V^\pi } = \sum\limits_{t = 0}^\infty  {{{({\gamma}{P^\pi })}^t}{r^\pi }}$.
```
would be clearer to separate the exponent of gamma and P, i.e., \gamma^t {P^\pi}^t

EvZissel replied:
Hi Aviv @avivt , I've separated the exponent as you suggested

avivt replied:
what I meant is \gamma^t {P^\pi}^t instead of (\gamma {P^\pi})^t

EvZissel replied:
@avivt I removed the parenthesis: commit ebf9eb5edc1031c1cd049a6d853d3a676adcd7d9

EvZissel committed:
Merge pull request #2 from avivt/master

Aviv sync

EvZissel committed:
Lecture 3 - revert change  

undo change - BF algorithm to use auxiliary array

EvZissel committed:
Lecture 5 - separate the exponent of gamma and P

line 125  - separate the exponent of gamma and P

EvZissel committed:
Lecture 5 - removed subscript from DP operator 

lines 372 and 410: for consistency with the notation in this chapter, I suggest to remove the gamma subscript from T (the gamma subscript only appears here).
line 453: missing whitespace

EvZissel committed:
Merge pull request #3 from avivt/master

Sync with avivt

EvZissel committed:
Lecture 5 - removed parenthesis in line 125

per https://github.com/avivt/046203-RL-lectures-notes/pull/14#discussion_r426375483
>what I meant is \gamma^t {P^\pi}^t instead of (\gamma {P^\pi})^t

avivt commented:
Thanks!
